docs/custom-types.md

```diff
@@ -0,0 +1,74 @@
+## Create custom property types
+
+While the Category Bundle comes with many build-in property types, your project might need one that is too specific to
+your use case than it could be added to the Category Bundle. Don't worry, it allows you to create your own property type
+that can be used in your project without contributing to the Bundle.
```

Comment:
Maybe we should rephrase this a bit, as the first sentence is hard to read.

Maybe something like?
```suggestion
While the Category Bundle comes with many build-in property types, your project might need one that is very specific to your use case. Adding such a specific type to the official bundle might not make sense. Don't worry, the bundle allows you to create your own property type that can be used in your project without contributing to the Bundle.
```